### PR TITLE
chore: 🤖 remove stale tests for standard flag and instances

### PR DESCRIPTION
**Pull request by anshumanv**
**What kind of change does this PR introduce?**
Cleanup instances of standard flag, not supported anymore

**Did you add tests for your changes?**
Yes

**If relevant, did you update the documentation?**
Yes

**Summary**
- standard flag is no longer supported, remove it's instances.

**Does this PR introduce a breaking change?**
No

**Other information**

### CODE REVIEW

File: test/standard/standard.test.js [hunk 1]
```diff
@@ -1,13 +0,0 @@
-const { run } = require('../utils/test-utils');
-
-describe('standard output', () => {
-    it('should print standard output', () => {
-        const { stdout, stderr } = run(__dirname);
```
Comment (snitin315) [inline test/standard/standard.test.js:5]: I think this test is not for `--standard` flag but the default webpack output (referred to as standard). Please correct me if I'm wrong.

Comment (anshumanv) [reply test/standard/standard.test.js:5]: No, this was used for --standard, but later --standard was dropped from the test when flag was removed.

Comment (snitin315) [reply test/standard/standard.test.js:5]: So, do we have another test for default output or we don't need one?

Comment (anshumanv) [reply test/standard/standard.test.js:5]: zero-config

Comment (anshumanv) [reply test/standard/standard.test.js:5]: I've added the things missing there, from this test we removed.

**Comment by rishabh3112** [comment]
Let's wait for #1508 to be merged as that PR also has changes for plugin tests and these changes could be conflicting.

**Pull request merged**